extensions/amp-script/amp-script.md

```diff
@@ -97,9 +110,9 @@ An `amp-script` element can load a JavaScript file from a URL:
 ```
 
 {% call callout('Important', type='caution') %}
-`amp-script` elements that use local scripts or cross-origin `src` require adding a `<meta name="amp-script-src" content="sha384-HASH">` to the document head, where `HASH` is a base64-encoded SHA-384 of the script source (similar to CSP's [`script-src`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#Sources)).
+`amp-script` elements that have a `script` or cross-origin `src` attribute require a ["script hash"](#security-features). Script hashes are specified in a `<meta name="amp-script-src" content="...">` element in the document head.
```

Comment:
One nit - would you mind updating the tips to the [new shortcode format](https://amp.dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting/?format=websites#tips)?